src/dex-ui/services/MirrorNodeService/MirrorNodeService.ts

```diff
@@ -104,6 +105,15 @@ function createMirrorNodeService() {
     ]);
   };
 
+  /**
+   * Fetches information related to a specific pair pair token.
+   * @param pairAddress  - The ID / Addresss of the pair token account to return data for.
+   * @returns Attributes associated with the provided token ID.
+   */
+  const fetchTokenPairsContract = async (pairAddress: string): Promise<MirrorNodeTokenPairResponse> => {
```

Comment:
Its simple gives contract information based on evm address. Pls rename to reflect that. Above you are using in the context of pair.

Reply:
fixed, changed it to `fetchContract`.